content/articles/deleting-domain.markdown

```diff
@@ -30,7 +39,3 @@ You should not delete an active domain registered or resolved with DNSimple.
 When a domain is hosted, DNSimple will stop resolving DNS for the domain.
```

Comment:
This looks like a repeat of lines 18 - 21, is that on purpose?

Reply:
It appears to be redundant to me, and I'm for removal.